### ISSUE TITLE
New Releases cheking/reporting thread

### ISSUE THREAD

**Issue by NEO-SPECTRUMAN** [Skulltrail192/One-Core-API-Binaries#55]
looks like some regression in 2.3.0 APP BASE
Qt 5.10.x based Grabber was worked up to 6.0.6 with gletches in 2.2.1
but in 2.3.0 there is crash only

**Comment by NEO-SPECTRUMAN** [comment]
also last XP ccleaner 5.64 start but crashes on 2.3.0
(i'm checked only on 2.3.0)

**Comment by NEO-SPECTRUMAN** [comment]
![2022-06-02_055335](https://user-images.githubusercontent.com/88002487/172067093-aa69a179-116e-4cf1-a9d0-e6e3e0cff1ca.png)

**Comment by Skulltrail192** [comment]
> looks like some regression in 2.3.0 APP BASE Qt 5.10.x based Grabber was worked up to 7.1.1 with gletches in 2.2.1 but in 2.3.0 there is crash only after instaling old one-core-api-app-base.exe only it runs/works again and vice versa

I think i was fixed

**Comment by NEO-SPECTRUMAN** [comment]
> I think i was fixed

nop
i tried install over 2.4.0
over clear sp3 en
over clear ncore (sv2k3)
and it without changes

or I doing it wrong...

**Comment by Skulltrail192** [comment]
About Grabber?

**Comment by Skulltrail192** [comment]
Is same problem of https://github.com/Skulltrail192/One-Core-API-Binaries/issues/41?

**Comment by NEO-SPECTRUMAN** [comment]
also if I installing kernel standalone
comparability is decreasing sharply

qt based software
![2022-06-20_081357](https://user-images.githubusercontent.com/88002487/174671014-fda04d03-a6b8-4e65-aef4-be4378ed5cf0.png)
![Копия 2022-06-20_071733](https://user-images.githubusercontent.com/88002487/174671187-4c5e9590-a164-4ffd-ad0b-90070f70e796.png)

**Comment by NEO-SPECTRUMAN** [comment]
> About Grabber?

about all tested noXP software

**Comment by NEO-SPECTRUMAN** [comment]
also after 2.0
there is NO kernel (without standalone) updates
that is how it should be?

**Comment by Skulltrail192** [comment]
I used steps: 
Case Base Installer first Base Installer -> New Installer (Additional Dlls) -> D3d Installer -> API-SET installer -> App Compat
And as i demonstred in https://github.com/Skulltrail192/One-Core-API-Binaries/issues/41#issuecomment-1154165673
I tested in Server 2003, i will test on XP. However, how i said, i deleted dwrite.dll to work.

**Comment by NEO-SPECTRUMAN** [comment]
does you reboot after install each package?
or after installing all is enough?

**Comment by Skulltrail192** [comment]
> does you reboot after install each package? or after installing all is enough?

Only base Installer and New Installer (Additional Dlls) require restart. 

I installed on fresh Windows Server 2003 install.

**Comment by NEO-SPECTRUMAN** [comment]
> I installed on fresh Windows Server 2003 install.

it works without 3D acceleration?

**Comment by Skulltrail192** [comment]
I transformed into workstation and enabled 3d acceleration, on Vmware.

**Comment by NEO-SPECTRUMAN** [comment]
i take new fresh english XP sp3 (without mods)
with
Base -> New -> D3d -> API-SET -> App Compat
and looks like the same result
![2022-06-20_235556](https://user-images.githubusercontent.com/88002487/174687655-f0ac52b3-2f09-42c9-ba7b-744cfdd0bb2d.png)

**Comment by NEO-SPECTRUMAN** [comment]
also in 2.5.0
doesn't run VC1719_redist.XPx86.exe
(Microsoft Visual C++ 2015-2019 Redistributable (x86) - 14.28.29213)
and I can test some software
previous 2 attempts (with 2.5.0) I installed it before
dependet software (like grabber) tried to run
but with error like on screenshot above

**Comment by NEO-SPECTRUMAN** [comment]
> I transformed into workstation and enabled 3d acceleration, on Vmware.

can you test vmware without 3d acceleration?

a tried virtualbox with it
but I suspect it may be virtualbox problems
now I'm don't risk to setup OCAPI on main system
because its very unstable
(and i lost about 20 "virtual" XP/sv2k3 with it :) )

**Comment by Skulltrail192** [comment]
Man, i think you don't deleted dwrite.dll. If this dll is present, both sonic and grabber not work.

**Comment by NEO-SPECTRUMAN** [comment]
> Man, i think you don't deleted dwrite.dll. If this dll is present, both sonic and grabber not work.

i thinked it will run with glechs as in ~2.2.0
and with deleting it will run without gliches

i will try to delete it later

**Comment by Skulltrail192** [comment]
> > Man, i think you don't deleted dwrite.dll. If this dll is present, both sonic and grabber not work.
> 
> i thinked it will run with glitches as in ~2.2.0 and after deleting it will run without glitches
> 
> i will try to delete it later

Yes, only some dialogs has problems, need investigation

**Comment by hlizard** [comment]
Chinese XP sp3 with 2.5.0(App Compat -> Base -> New -> D3d -> API-SET):
vscode 1.18 no any window display (can display window with 2.2.1, but crash when click "File" menu)
firefox 54 crash when input
notepad++ 8.4.2 no open/save dialog display

**Comment by NEO-SPECTRUMAN** [comment]
so i tried delete dwrite.dll in system32
and it works for some software
but VCRedist doesn't instaling
I installed BASE from previous 2.4.0
installed vcredist
![2022-06-28_110718](https://user-images.githubusercontent.com/88002487/176130064-410bfdda-72ff-45b0-8063-4ecb1cb1287a.png)
Installed again BASE from 2.5.0
and it works too

**Comment by NEO-SPECTRUMAN** [comment]
also I tryed firefox60
it doen't works

**Comment by mina354** [comment]
I have some bugs to report:

In Power Options, Hibernate function is removed.
In general, Windows Powershell (and ISE too) doesn't run.
In Add or Remove Programs, at Set Program Access and Defaults, the 3rd button on the bottom right is missing text (which is, "Help")

All of the above bugs happen only when One-Core-API is installed. They don't happen when One-Core-API isn't installed. Hopefully they will be fixed in the future.

@Skulltrail192


@NEO-SPECTRUMAN 
> also I tryed firefox60 it doesn't works

Any Firefox version above 54 doesn't work.

@hlizard


> firefox 54 crash when input notepad++ 8.4.2 no open/save dialog display

The Firefox issue is a known issue,

As for Notepad++, since 7.9.3, the new open/save dialogs are used, and XP support dropped in that version. Hopefully, this project makes open/save dialogs display again. This also occurs in Firefox since version 53.

**Comment by Skulltrail192** [comment]
> In Add or Remove Programs, at Set Program Access and Defaults, the 3rd button on the bottom right is missing text (which is, "Help")

In Add or Remove Programs, at Set Program Access and Defaults, the 3rd button on the bottom right is missing text (which is, "Help") 
On all my Windows XP or Windows Server 2003 Systems, befeore any one core package, i don't text in this button. This is not a bug.

**Comment by Skulltrail192** [comment]
> also I tryed firefox60 it doesn't works

I don't said Firefox 60 works. Only until 54 works.

**Comment by mina354** [comment]
> i just report that firefox60 doesn't work &)
> 
> there is no any guaranty that any "above" version automatically will doesn't work

The release notes of version 2.4.0 did mention that firefox 55+ don't work.

**Comment by Skulltrail192** [comment]
> > i just report that firefox60 doesn't work &)
> > there is no any guaranty that any "above" version automatically will doesn't work
> 
> The release notes of version 2.4.0 did mention that firefox 55+ don't work.

On version 2.4.0 release notes has this text:
"Known issues:"
"Firefox version 55 or above not working;" 
So it is NOT working

**Comment by Skulltrail192** [comment]
> > Any Firefox version above 54 doesn't work.
> 
> > I don't said Firefox 60 works. Only until 54 works.
> 
> i just report that firefox60 doesn't work &)
> 
> there is no any guaranty that any "above" version automatically will doesn't work
> 
> before i mentioned 2 software that doesn't works "above" some version and works "above" "above" some version again and doesn't works "above" "above" "above" again...

On version 2.4.0 release notes has this text:
"Known issues:"
"Firefox version 55 or above not working;"
So it is NOT working

**Comment by NEO-SPECTRUMAN** [comment]
:facepalm:

**Comment by mina354** [comment]
@NEO-SPECTRUMAN What's your problem dude?? We already told you that Firefox version 55 or above don't work, we even quoted the original text, what more do you want to believe it?

**Comment by NEO-SPECTRUMAN** [comment]
what about?
> Added --no-sandbox and --enable-exclusive-audio on CreateProcess to work without set these parameters on shortcut;

you can't put any additional arguments to all software
you will broke thousands of console apps

but i'm checking argc, argv
and there is no any extra arguments not in normal mod nor in compatibility mode

also win11 compatibility mode is fell off :)

**Comment by Skulltrail192** [comment]
> you can't put any additional arguments to all software you will broke thousands of console apps
> 
> but i'm checking argc, argv and there is no any extra arguments neither in normal mode nor in compatibility mode
> 
> how it works?

it was add for some exe files, has a verification. every cmd applications working fine. Don't worry, I do things right

**Comment by Skulltrail192** [comment]
> > you can't put any additional arguments to all software you will broke thousands of console apps
> > but i'm checking argc, argv and there is no any extra arguments neither in normal mode nor in compatibility mode
> > how it works?
> 
> it was add for some exe files, has a verification. every cmd applications working fine. Don't worry, I do things right

**Comment by NEO-SPECTRUMAN** [comment]
> For Palemoon, you just need delete api-ms-win-core-memory-l1-1-0.dll

also you must describe "hints" to run some software
in readme.md?
or it will be lost...
![2022-07-06_004747](https://user-images.githubusercontent.com/88002487/177422409-55ad0684-298a-48e0-92e8-1ed6b5388fff.png)

**Comment by mina354** [comment]
> I have some bugs to report:
> 
> In Power Options, Hibernate function is removed. In general, Windows Powershell ISE doesn't run. In Add or Remove Programs, at Set Program Access and Defaults, the 3rd button on the bottom right is missing text (which is, "Help")
> 
> All of the above bugs happen only when One-Core-API is installed. They don't happen when One-Core-API isn't installed. Hopefully they will be fixed in the future.

@Skulltrail192 How about these bugs? You can consider the first two bugs a bit major, but the last one is likely a text bug.

**Comment by mina354** [comment]
> also win11 compatibility mode is fell off :)

@NEO-SPECTRUMAN Not just Windows 11, also Windows 10 21H2. :(

**Comment by Skulltrail192** [comment]
> Powersh

I found solution to all issue. Avaliable on next release.

**Comment by mina354** [comment]
> > Powersh
> 
> I found solution to all issue. Avaliable on next release.

Good to hear. Thanks.

> > > also win11 compatibility mode is fell off :)
> > 
> > 
> > @NEO-SPECTRUMAN Not just Windows 11, also Windows 10 21H2. :(
> 
> It was removed on 2.6 because not working. Will be added again on some next release;

Dammit, knew it. Guess I'll just wait for the next releases.

**Comment by mina354** [comment]
@NEO-SPECTRUMAN if that makes you happy, the latest Firefox version to work on Windows XP with One-Core-API is Firefox Nightly 55.0a1 (2017-03-21). I used [this version.](http://ftp.mozilla.org/pub/firefox/nightly/2017/03/2017-03-21-03-02-11-mozilla-central/)

**Comment by mina354** [comment]
@Skulltrail192 Well, OCAPI 2.7.0 is supposed to fix Windows PowerShell ISE crash issue. But, it still persists, even in 2.7.0.
![image](https://user-images.githubusercontent.com/87821255/180031645-60efbcf8-6fb9-4f12-ad65-6973d141743c.png)
I used this installation order: App Compat -> Base -> Driver Update -> API-SET -> D3d -> Additional Dlls.
The other 2 issues (The Hibernate tab in Power Options disappearing, and the missing "Help" text on the Set Program Access and Defaults in Add or Remove Programs) are fixed in this version, though.

**Comment by Skulltrail192** [comment]
Are you uninstalled packages before?

**Comment by mina354** [comment]
> Are you uninstalled packages before?

No, I did not.

**Comment by mina354** [comment]
I also want to report another issue: FileZilla starts incorrectly like this everytime in XP SP3 with OCAPI installed:
![image](https://user-images.githubusercontent.com/87821255/180093308-a4385f92-fa2e-40ae-8274-860ece75371f.png)
While in Windows 7, it always starts correctly like this:
![image](https://user-images.githubusercontent.com/87821255/180093720-3ff6ebaa-323e-4f70-97c0-fc8a33f40e80.png)

**Comment by NEO-SPECTRUMAN** [comment]
looks like there is another SH*T
named 
VC_redist 2015-2019 14.27.29114+  (~14.27 last for XP)
VC_redist 2022 etc

and its can't be installed?
![2022-08-01_023534](https://user-images.githubusercontent.com/88002487/182051398-76f05e0f-1b54-47ac-9f9e-7cd4181098a0.png)

**Comment by NEO-SPECTRUMAN** [comment]
also last VC_redist for XP
can be found at https://github.com/abbodi1406/vcredist/issues/22

M$ says
![2022-08-01_023152](https://user-images.githubusercontent.com/88002487/182086775-d9d1c2ce-5084-4beb-a562-3cafdd06792b.png)
 
also this big links list
https://github.com/abbodi1406/vcredist/tree/master/source_links
says that last is 14.28.29213.0

**Comment by mina354** [comment]
Ok, it seems that the Windows PowerShell ISE crash has been fixed, but when i open it, it hangs like this:
![image](https://user-images.githubusercontent.com/87821255/183252656-00398037-5469-4389-99f4-a3a446e79eb8.png)

**Comment by Skulltrail192** [comment]
> looks like there is another SH*T named VC_redist 2015-2019 14.27.29114+ (~14.27 last for XP) VC_redist 2022 etc
> 
> and its can't be installed? ![2022-08-01_023534](https://user-images.githubusercontent.com/88002487/182051398-76f05e0f-1b54-47ac-9f9e-7cd4181098a0.png)

Well, Starting visual Studio 2013, some Microsoft application use a different way to verify compatbility. Even you set Windows Vista or above, it will not run. I think it is related with Manisfest what determine version of application can be performed, or it can check file info version, present on kernel32, for example

**Comment by Skulltrail192** [comment]
> Ok, it seems that the Windows PowerShell ISE crash has been fixed, but when i open it, it hangs like this: ![image](https://user-images.githubusercontent.com/87821255/183252656-00398037-5469-4389-99f4-a3a446e79eb8.png)

What version of Vmware used?

**Comment by mina354** [comment]
16.2.4

**Comment by Skulltrail192** [comment]
I think it is related with D3D. Older versions of Vmware (i use version 12) it's works fine.

**Comment by mina354** [comment]
> I also want to report another issue: FileZilla starts incorrectly like this everytime in XP SP3 with OCAPI installed: ![image](https://user-images.githubusercontent.com/87821255/180093308-a4385f92-fa2e-40ae-8274-860ece75371f.png)

Is there no solution on this?

**Comment by Skulltrail192** [comment]
For now, no. I Will try to next version.

**Comment by mina354** [comment]
> For now, no. I Will try to next version.

Ok, thanks. It's good to see you noting down bugs like this for later testing.

**Comment by mina354** [comment]
> I think it is related with D3D. (directx) Older versions of Vmware (i use version 12) it's works fine.

Luckily, it doesn't happen in VirtualBox.
